kolibri/plugins/learn/options.py

```diff
@@ -0,0 +1,9 @@
+option_spec = {
+    "LEARN": {
+        "DISABLE_CUSTOM_CHANNEL_NAV": {
+            "type": "boolean",
+            "default": False,
```

Comment:
The issue asks for this to be disabled by default, so if the flag is going to be `DISABLE` then this should default to `True`!

Reply:
Made it into an `ENABLE` name instead